tests/config/tailscale_key.yaml

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
```

Comment:
I think we need to replace the copyright with the Dapr one and then add theirs below. For theirs, include:
- A link to where the code comes from
- Their copyright (“Copyright (C) 2021 Tailscale Inc & AUTHORS. All rights reserved”)
- A mention of the license and a link to the LICENSE file that’s mentioned there

Reply:
could you check now?

Reply:
This does not make sense. This is Dapr specific.

Reply:
@artursouza is right on this file. The other ones seem copied from https://github.com/tailscale/tailscale/tree/main/docs/k8s but this one is original from us, right? So no need to keep their copyright on this specific file.